backend/src/main/proto/gtfs-realtime.proto

```
@@ -0,0 +1,1259 @@
+//
+// GTFS Realtime lets transit agencies provide consumers with realtime
+// information about disruptions to their service (stations closed, lines not
+// operating, important delays etc), location of their vehicles and expected
+// arrival times.
+//
+// This protocol is published at:
+// https://github.com/google/transit/tree/master/gtfs-realtime
+
+syntax = "proto2";
+option java_package = "com.google.transit.realtime";
+package transit_realtime;
```

Buf (1.65.0)

[error] Files with package "transit_realtime" must be within a directory "transit_realtime" relative to root but were in directory "backend/src/main/proto".

(PACKAGE_DIRECTORY_MATCH)